src/sagemaker/cli/compatibility/v2/modifiers/serde.py

```diff
@@ -0,0 +1,407 @@
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Classes to modify serializer and deserializer code to be compatible with
+version 2.0 and later of the SageMaker Python SDK.
+"""
+from __future__ import absolute_import
+
+import ast
+
+import pasta
+
+from sagemaker.cli.compatibility.v2.modifiers import matching
+from sagemaker.cli.compatibility.v2.modifiers.modifier import Modifier
+
+OLD_AMAZON_CLASS_NAMES = {"numpy_to_record_serializer", "record_deserializer"}
+NEW_AMAZON_CLASS_NAMES = {"RecordSerializer", "RecordDeserializer"}
+
+# The values are tuples so that the object can be passed to matching.matches_any.
+OLD_CLASS_NAME_TO_NAMESPACES = {
+    "_CsvSerializer": ("sagemaker.predictor",),
+    "_JsonSerializer": ("sagemaker.predictor",),
+    "_NpySerializer": ("sagemaker.predictor",),
+    "_CsvDeserializer": ("sagemaker.predictor",),
+    "BytesDeserializer": ("sagemaker.predictor",),
+    "StringDeserializer": ("sagemaker.predictor",),
+    "StreamDeserializer": ("sagemaker.predictor",),
+    "_NumpyDeserializer": ("sagemaker.predictor",),
+    "_JsonDeserializer": ("sagemaker.predictor",),
+}
```

Comment:
you could also use a comprehension here since these all have the same namespace

Reply:
Fixed